merge_test.go

```diff
@@ -1767,7 +1886,7 @@ func TestMergeArray(t *testing.T) {
 				So(err, ShouldBeNil)
 			})
 
-			Convey("throw an error when delete point cannot be found", func() {
+			Convey("don't throw an error when delete point cannot be found", func() {
```

Comment:
I have added a test for some characters +,. and *. 

So do we agree on going with this caveat or should I implement something like `(( delete regex "dea.*" ))`?